fcrepo-http-api/src/main/java/org/fcrepo/http/api/Transactions.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.http.api;
+
+// import static java.util.Date.from;
+// import static javax.ws.rs.core.Response.created;
+// import static javax.ws.rs.core.Response.noContent;
+// import static javax.ws.rs.core.Response.status;
+// import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
+import static org.slf4j.LoggerFactory.getLogger;
+
+// import java.net.URI;
+import java.net.URISyntaxException;
+
+// import javax.inject.Inject;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.core.Response;
+
+// import org.fcrepo.kernel.api.TransactionManager;
+import org.slf4j.Logger;
+import org.springframework.context.annotation.Scope;
+
+/**
+ * Transactions over REST
+ *
+ * @author awoods
+ * @author gregjan
+ */
+@Scope("prototype")
+@Path("/{path: .*}/fcr:tx")
+public class Transactions extends FedoraBaseResource {
```

Comment:
Create a JIRA for re-implementing the Transactions endpoint.

Reply:
https://jira.duraspace.org/browse/FCREPO-3074